tribler.sh

```diff
@@ -27,7 +27,8 @@ if [ "$UNAME" = "Linux" ]; then
     }
 
     python2.7 $TRIBLER_SCRIPT
-
+elif [[ "$UNAME" = CYGWIN_NT* ]]; then
+    python $TRIBLER_SCRIPT
```

Comment:
@whirm in the 3rd example you're checking against the string "HEL*" I think. Removing the quotes around `HEL*` works for me.